src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        StringBuilder employeeSalary = new StringBuilder();
+        employeeSalary.append("Report for period ").append(dateFrom)
+                .append(" - ").append(dateTo).append("\n");
+        LocalDate dateStart = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate dateFinish = LocalDate.parse(dateTo, FORMATTER);
+        for (String name : names) {
+            int totalSalary = 0;
+            for (String information1 : data) {
```

Comment:
it's recommended to make naming different from information vs information1 ,  it's hard to see a difference right away